docs/contributing.md

```diff
@@ -63,46 +64,61 @@ GLOBAL OPTIONS:
-is not specified then keybaseca will accept messages in any channel of any team listed in the `TEAMS` environment variable.
-All communication happens via the Go chatbot library. 
-
-Prior to sending a `SignatureRequest`, kssh sends a series of `AckRequest` messages. An `AckRequest` message is sent until 
-kssh receives an `Ack` from keybaseca. This is done in order to ensure that kssh has correctly connected to the chat channel
-and that the bot is responding to messages. Afterwards, a `SignatureRequest` packet is sent and keybaseca parses it and 
-returns a signed key. Note that only public keys and signatures are sent over Keybase chat and private keys never 
-leave the devices they were generated on. 
+kssh and keybaseca communicate with each other over Keybase chat. If the
+`CHAT_CHANNEL` environment variable is specified in keybaseca's environment,
+keybaseca will only accept communication in the specified team and channel.
+This configuration is passed to kssh clients via the client configs stored in
+the [KV store](https://keybase.io/docs/bots/kvstore).  If the `CHAT_CHANNEL`
+environment variable is not specified then keybaseca will accept messages in
+any channel of any team listed in the `TEAMS` environment variable.  All
+communication happens via the Go chat bot library. 
+
+Prior to sending a `SignatureRequest`, kssh sends a series of `AckRequest`
+messages. An `AckRequest` message is sent until kssh receives an `Ack` from
+keybaseca. This is done in order to ensure that kssh has correctly connected to
+the chat channel and that the bot is responding to messages. Afterwards, a
+`SignatureRequest` packet is sent and keybaseca parses it and returns a signed
+key. Note that only public keys and signatures are sent over Keybase chat and
+private keys never leave the devices they were generated on. 
 
 #### SSH Operations
 
-When the ssh-keygen command is available, ssh keys are generated via the ssh-keygen command. In this case, generated
-SSH keys are ed25519 keys. If the ssh-keygen command is not available, SSH keys are generated in pure go code and are 
-ecdsa keys. 
+When the ssh-keygen command is available, ssh keys are generated via the
+ssh-keygen command. In this case, generated SSH keys are ed25519 keys. If the
+ssh-keygen command is not available, SSH keys are generated in pure go code and
+are ecdsa keys. 
 
-keybaseca uses the ssh-keygen binary in order to complete all key signing operations. 
+keybaseca uses the ssh-keygen binary in order to complete all key signing
+operations. 
 
 #### KBFS
 
-In order to ensure that keybaseca can run inside of docker (which does not support FUSE filesystems without adding
-the CAP_SYS_ADMIN permission), all KBFS interactions are done via `keybase fs ...` commands. This makes it so that 
+In order to ensure that keybaseca can run inside of docker (which does not
+support FUSE filesystems without adding the CAP_SYS_ADMIN permission), all KBFS
```

Comment:
is this still necessary?

Reply:
added this sentence at the beginning of the paragaph: `"keybaseca supports logging to a local or KBFS file. "`